r-package/grf/NAMESPACE

```diff
@@ -27,8 +27,11 @@ export(quantile_forest)
 export(regression_forest)
 export(split_frequencies)
 export(test_calibration)
+export(tune_causal_forest)
+export(tune_forest)
```

Comment:
I don't think `tune_forest` should be exported, it seems like part of the internal implementation.